### PR TITLE
Prevent waiting time on svirt console commands and grub

### PR DESCRIPTION
**Pull request by okurz**
* Explicitly confirm grub boot selection in svirt console to save some seconds
* Extract helper function for typing on svirt console and waiting for answer
* Fix wait_serial waiting on svirt commands never sent

### CODE REVIEW

**Comment by okurz** [comment]
ah, don't merge. needs tidy

File: lib/opensusebasetest.pm [hunk 1]
```diff
@@ -271,18 +271,14 @@ Also see poo#18016 for details.
...
 sub rewrite_static_svirt_network_configuration {
     my ($self) = @_;
-    type_string "echo root > \$pty\n";
-    wait_serial('Password');
-    type_string "echo $testapi::password > \$pty\n";
+    type_line_svirt "root", expect => 'Password';
+    type_line_svirt "$testapi::password";
     my $virsh_guest = get_required_var('VIRSH_GUEST');
-    type_string "echo sed -i \"\\\"s:IPADDR='[0-9.]*/\\([0-9]*\\)':IPADDR='$virsh_guest/\\1':\\\" /etc/sysconfig/network/ifcfg-\*\" > \$pty\n";
+    type_line_svirt "sed -i \"\\\"s:IPADDR='[0-9.]*/\\([0-9]*\\)':IPADDR='$virsh_guest/\\1':\\\" /etc/sysconfig/network/ifcfg-\*\"", expect => '#';
```
Comment (Soulofdestiny) [inline lib/opensusebasetest.pm]: did you crosscheck if this escaping hell still works ? :)

Comment (okurz) [inline lib/opensusebasetest.pm]: yes, works -> http://lord.arch/tests/7099#step/boot_to_desktop_sym/15

**Comment by okurz** [comment]
tidied

**Pull request merged**